bindings/python/malamute/_malamute_ctypes.py

```diff
@@ -7,7 +7,7 @@
 import os, sys
 from ctypes import *
 from ctypes.util import find_library
-import czmq
+import czmq.git
```

Comment:
@bluca @vyskocilm I'm not able to recreate this issue. The property that is used for imports is $(dependency.project) which is fine in my case. For whatever reason the property seems to be corrupted here.